<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FSist</class>
 <widget class="QWidget" name="FSist">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1036</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Журнал работы</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Отбор по датам</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Начало:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="dtn">
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Конец:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="dtk">
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Отбор по результатам:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbRez">
           <item>
            <property name="text">
             <string>все</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>выполнено</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>не выполнено</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pbRefresh">
     <property name="text">
      <string>Обновить</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/res/Sync.png</normaloff>:/res/Sync.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
